### ISSUE TITLE
Allow attachments removal

### ISSUE THREAD

**Issue by PeopleInside** [uvdesk/community-skeleton#463]
**Description**  

Please allow authorized operators to remove attachments from a ticket.
Customer can upload big attachments r something that should be not attached to a ticket.

Currently seems operator or admin cannot do nothing about attachments.

**Comment by vipin-shrivastava** [comment]
@PeopleInside 

 If you delete a thread then attachment also deleted with that

**Comment by PeopleInside** [comment]
Yes but I don't have to loose the full ticket, just I need remove the wrong attachments and cannot do.

**Comment by vipin-shrivastava** [comment]
@PeopleInside 

Thanks,  we will try to provide this feature from next release
